spec/image_spec.rb

```diff
@@ -176,7 +176,7 @@
         generate_test_image(input_path, color: 'cmyk', cg_type: 'cmyk', profile: 'cmyk', bands: 4)
       end
 
-      it 'creates an srgb jp2', skip: 'Need to verify the color space is correct in jp2' do
+      it 'creates an srgb jp2' do
```

Comment:
One concept of this test is that a CMYK image becomes an sRGB image for the jp2.  We can test the temporary tiff instead of the jp2 to figure this out.

expect(Vips::Image.new_from_file([temporary tiff created]).get_value('interpretation')).to eq :srgb

this requires exposing the temporary tiff enough to be tested this way.